lib/matplotlib/axis.py

```diff
@@ -641,6 +643,10 @@ def __init__(self, axes, *, pickradius=15):
         pickradius : float
             The acceptance radius for containment tests. See also
             `.Axis.contains`.
+        clear : bool, default: True
+            Whether to clear the Axis on creation. This is not required, e.g.,  when
+            creating an Axis as part of an Axes, as ``Axes.clear`` will call
+            ``Axis.clear``.
```

Comment:
Not possible to link as there are three `Axes.clear` methods and I wanted to clearly state the difference between `Axes.clear` and `Axis.clear`.